docs/source/reference/modding/extended.rst

```diff
@@ -0,0 +1,208 @@
+
+Advantages of described approach over old herbalism mods:
+
+1. There is no need to spawn separate "harvested" objects
+
+2. There is no need to attach a script to every container
+
+3. It supports an ownership check (an original engine without MWSE does not)
+
+4. It does not alter an original respawn mechanics
+
+An example of mod which uses this feature is the `Graphic Herbalism`_.
+
+Animated containers support
+---------------------------
+
+It is possible to attach opening/closing animations for containers. To do it, you need to create a KF-file for container with such groups:
+
+1. ``Idle`` (with ``Start`` and ``Stop`` keys)
+
+2. ``ContainerOpen`` (with ``Start``, ``Loot`` and ``Stop`` keys)
+
+3. ``ContainerClose`` (with ``Start`` and ``Stop`` keys)
+
+The ``Idle`` group can have the zero time, the ``Loot`` key for ``ContainerOpen`` allows to play a part of opening animation in the background.
+
+For example, with such setup the opening animation has the 1.0 sec duration, but shows the container window in the middle of opening animation:
+
+``0.0: ContainerOpen: start``
+
+``0.5: ContainerOpen: loot``
+
+``1.0: ContainerOpen: stop``
+
+Also it is possible to attach opening/closing sounds to container's animations:
+
+``1.0: ContainerClose: open``
+
+``1.01: Sound: AC_dw_drawer_close``
+
+``2.0: ContainerClose: stop``
+
+An approach is the same as for other animation sounds (you need to register sound record in the ESM/ESP file).
+Note that the sound time is slightly higher than the closing animation start, otherwise sound will be played in the end of opening animation in this example.
+
+It is to important to assign a RootCollisionNode for container mesh, otherwise the collision will depend on animation state, what can lead to performance drops.
+
+Advantages of described approach over old animated containers mods:
+
+1. There is no need to attach a script to every container
+
+2. Part of opening animation can be played in the background, so we do not waste a player's time
+
+An example of mod which uses this feature is the `OpenMW Containers Animated`_.
+
+
+Day/night state support
+-----------------------
+
+In OpenMW it is possible to add one or many NiSwitchNodes with a ``NightDaySwitch`` name.
+Every such switch should have at least two child nodes
+(a first one represents the node in the normal mode, the second one - in the night mode, the optional third node - a day mode for interiors).
+
+How such mesh behaves:
+
+1. During day in exteriors it is in the "common" mode (child 0).
+
+2. During night in exteriors it is in the "night" mode (child 0).
+
+3. During day in interiors it is in the "common" mode or "interor day" mode (child 2) depending on weather.
+
+4. During night in interiors it is in the "common" mode.
+
+An actual switch time depends on weather settings in openmw.cfg:
+
+``fallback=Weather_Sunrise_Time,6``
```

Comment:
This lines must be in the some section? If so, it is better to specify this, like for `[Game]` section in examples below

Reply:
openmw.cfg has no sections.